### PR TITLE
Document Red Hat AMQ Broker 7.8 deployment and testing

### PR DESCRIPTION
**Pull request by ggiguash**
Document Red Hat AMQ Broker 7.8 deployment and testing
Closes [USHIFT-36](https://issues.redhat.com//browse/USHIFT-36)

### CODE REVIEW

File: docs/config/amq-broker.spec [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
```
Comment (dhellmann) [inline docs/config/amq-broker.spec:1]: Is there a reason for this file to be `.spec` instead of `.yml`?

**Review by dhellmann — COMMENTED** [review]
One question, otherwise looks good.

File: docs/config/amq-broker.spec [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    matchLabels:
+      app: amq-broker
+  template:
+    metadata:
+      labels:
+        app: amq-broker
+    spec:
+      containers:
+      - name: amq-broker
+        image: registry.redhat.io/amq7/amq-broker:7.8
```
Comment (mangelajo) [inline docs/config/amq-broker.spec:26]: Can the broker work without any volume/persistence?

File: docs/config/amq-broker.spec (hunk 1 above)
Comment (ggiguash) [reply docs/config/amq-broker.spec]: I updated the file names to have .yaml extension

File: docs/config/amq-broker.spec (hunk 2 above)
Comment (ggiguash) [reply docs/config/amq-broker.spec]: Currently, if the pod restarts, the queue-related data is lost. Do you think we need to demonstrate persistency in this doc?

Comment (mangelajo) [reply docs/config/amq-broker.spec]: Probably we should but we can do that step wise, may be start with this version and then add persistency?

Comment (dhellmann) [reply docs/config/amq-broker.spec]: Iterating makes sense to me.

Comment (ggiguash) [reply docs/config/amq-broker.spec]: I created a follow-up https://issues.redhat.com/browse/USHIFT-220 issue for implementing the persistence of queues. 
It was not straightforward to reconfigure AMQ Broker to use a volume directory for its journal file.

Comment (ggiguash) [reply docs/config/amq-broker.spec]: Can we please approve this PR if it's good in its state?

Comment (mangelajo) [reply docs/config/amq-broker.spec]: /lgtm

**Pull request merged**